FileServer/FileService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using FileSignatures;
+
+namespace FileServer
+{
+    public class FileService
+    {
+        public string? IsValidFileExtensionAndSignature(string fileName, Stream data, string[] permittedExtensions)
```

Comment:
Hàm là bắt đầu bằng `Is` thì nên trả về kiểu boolean nhé

Reply:
Những cái lỗi bên dưới t nghĩ throw error luôn cũng được, xong có gì xử lý vào trong `ExceptionFilter` (trả về lỗi 400 nhé)

Reply:
Đã sửa function name

Reply:
ok